test/TestGrains/StatelessWorkerGrain.cs

```diff
@@ -60,14 +59,12 @@ private static Task TimerCallback(object state)
         }
 
 
-        public Task<Tuple<Guid, List<Tuple<DateTime, DateTime>>>> GetCallStats()
+        public Task<Tuple<Guid, string, List<Tuple<DateTime, DateTime>>>> GetCallStats()
         {
             Thread.Sleep(200);
-            lock (allActivationIds)
```

Comment:
oops, yeah, while I was troubleshooting the forwarding failure I avoid any locks by using concurrent collections, but then I reverted as that wasn't the cause... I guess I forgot to revert this. It's not harmful for these tests, since the hashset is not mutated once we start getting the stats, but I'll still revert it.